.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
             pip install --upgrade setuptools
             cd ml-agents-envs && pip install -e .
             cd ../ml-agents && pip install -e .
-            pip install black pytest-cov==2.6.1 codacy-coverage==1.3.11
+            pip install pre-commit pytest-cov==2.6.1 codacy-coverage==1.3.11
```

Comment:
pre-commit will install all the corresponding hooks that it needs, so we don't need to install black directly anymore.